javascript/house-style.md

```diff
@@ -195,6 +195,46 @@ We don't do this:
 const uniqueIDs = new Set([1, 2, 1, 2]);
 ```
 
+#### JS Doc
+
+Although your JavaScript code should be self-documenting, we highly recommend
+that you to document your JavasScript application or library using the
+Standardised [JS Doc](https://jsdoc.app/) format.
+
+This format is leveraged by the Language Server Protocol (LSP), which is
+supported in numerous editors and Integrated Development Environments (IDE)
+nowadays.  
+Thanks to LSP you get short lines of documentation right from the
+invocation context. This helps you code quickly, while avoiding obvious errors.
+
+For functions for instance, you get:
```

Comment:
Consider using e.g. for example instead of for instance since instance has a touch of ambiguity in that context:

```suggestion
For e.g. functions, you get:
```